airbyte-integrations/connectors/destination-kafka/src/main/resources/spec.json

```diff
@@ -191,62 +191,62 @@
         "type": "integer",
-        "default": 1048576
+        "examples": [1048576]
       },
       "retries": {
         "title": "Retries",
         "description": "Setting a value greater than zero will cause the client to resend any record whose send fails with a potentially transient error.",
         "type": "integer",
-        "default": 2147483647
+        "examples": [2147483647]
       },
       "socket_connection_setup_timeout_ms": {
         "title": "Socket connection setup timeout",
         "description": "The amount of time the client will wait for the socket connection to be established.",
-        "type": "number",
-        "default": 10000
+        "type": "string",
+        "examples": [10000]
       },
       "socket_connection_setup_timeout_max_ms": {
         "title": "Socket connection setup max timeout",
         "description": "The maximum amount of time the client will wait for the socket connection to be established. The connection setup timeout will increase exponentially for each consecutive connection failure up to this maximum.",
-        "type": "number",
-        "default": 30000
+        "type": "string",
+        "examples": [30000]
       },
       "max_block_ms": {
         "title": "Max block ms",
         "description": "The configuration controls how long the KafkaProducer's send(), partitionsFor(), initTransactions(), sendOffsetsToTransaction(), commitTransaction() and abortTransaction() methods will block.",
-        "type": "number",
-        "default": 60000
+        "type": "string",
+        "examples": [60000]
       },
       "request_timeout_ms": {
         "title": "Request timeout",
         "description": "The configuration controls the maximum amount of time the client will wait for the response of a request. If the response is not received before the timeout elapses the client will resend the request if necessary or fail the request if retries are exhausted.",
         "type": "integer",
-        "default": 30000
+        "examples": [30000]
       },
       "delivery_timeout_ms": {
         "title": "Delivery timeout",
         "description": "An upper bound on the time to report success or failure after a call to 'send()' returns.",
         "type": "integer",
-        "default": 120000
+        "examples": [120000]
       },
       "send_buffer_bytes": {
         "title": "Send buffer bytes",
         "description": "The size of the TCP send buffer (SO_SNDBUF) to use when sending data. If the value is -1, the OS default will be used.",
         "type": "integer",
-        "default": 131072
+        "examples": [131072]
       },
       "receive_buffer_bytes": {
         "title": "Receive buffer bytes",
         "description": "The size of the TCP receive buffer (SO_RCVBUF) to use when reading data. If the value is -1, the OS default will be used.",
         "type": "integer",
-        "default": 32768
+        "examples": [32768]
```

Comment:
why not keep the default?

Reply:
Right now Default value in spec.json is not being sent to API request => change the default to examples and force user to insert value.

Reply:
I'll create another issue to investigate this in the front-end.